micro-rdk/src/common/webrtc/grpc.rs

```diff
@@ -20,6 +20,12 @@ use crate::{
 
 use super::{api::WebRtcError, sctp::Channel};
 
+#[cfg(feature = "camera")]
+// sizeof(fake_image) + 30bytes for headers
+static WEBRTC_GRPC_BUFFER_SIZE: usize = 10492;
```

Comment:
confirmed as minimum threshold with python and go sdk. May need to bump for `Message`s with larger encodings/headers